WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift

```diff
@@ -95,6 +95,16 @@ public struct WordPressAuthenticatorConfiguration {
     /// Default value is disabled
     let continueWithSiteAddressFirst: Bool
 
+    /// If enabled shows a "Sign in with site credentials" button in `GetStartedViewController` when landing in the screen after entering site address
+    ///  Used in WooCommerce iOS to enable sign in using site credentials feature.
```

Comment:
Nit: I think we may not need to include that this is used for WCiOS, as any app should be able to use it. Let reword this to something more generic:
```suggestion
    ///  Used to enable sign-in to self-hosted sites using WordPress.org credentials.
```

Reply:
Done in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/649/commits/98e80ab0828e6c3398fb7735e56200f7dc5e714f